tomls/omnibus-base-mainnet-andromeda/perps/arb-invokes.toml

```diff
@@ -0,0 +1,157 @@
+include = [
```

Comment:
please make sure to remove any nested includes and put them on the top level.
I have done it for testnet and they somehow appeared here for mainnet too...